src/operator/src/insert.rs

```diff
@@ -558,7 +604,7 @@ impl Inserter {
             }
         }
 
-        Ok(table_name_to_ids)
+        Ok((table_name_to_ids, instant_table_ids))
```

Comment:
Will we auto create an instant table in the future? Do we need to check this after L596?

Reply:
> Will we auto create an instant table in the future? Do we need to check this after L596?

I don't think we should auto create an instant table for it's a very unexpected behavior and should be set by user explictly anyway